code/datums/supplypacks/misc.dm

```diff
@@ -322,3 +320,66 @@
 	cost = 50
 	containertype = /obj/structure/closet/crate
 	containername = "Neon Lights crate"
+
+//Culture Crates
+/datum/supply_pack/misc/culture_apidean
+	name = "Apidean Culture Crate"
+	contains = list(
+			/obj/fiftyspawner/wax = 2,
+			/obj/item/healthanalyzer/apidean = 1,
+			/obj/item/analyzer/apidean = 1,
+			/obj/item/reagent_scanner/apidean = 1
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate
+	containername = "Apidean Culture crate"
+
+/datum/supply_pack/misc/weaponry_apidean
+	name = "Apidean Weaponry Crate"
+	contains = list(
+			/obj/item/gun/projectile/apinae_stinger = 1,
+			/obj/item/gun/projectile/apinae_pistol = 2,
+			/obj/item/grenade/spawnergrenade/manhacks/apidean = 3
+			)
+	cost = 100
+	containertype = /obj/structure/closet/crate/secure/gear
+	containername = "Apidean Weaponry crate"
+	access = access_security
+
+//WIP Needs more stuff.
+/datum/supply_pack/misc/culture_tyrmalin
+	name = "Tyrmalin Culture Crate"
+	contains = list(
+			/obj/item/tool/wrench/goblin = 2,
+			/obj/item/weldingtool/welder_spear = 2
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate
+	containername = "Tyrmalin Culture crate"
+
+/datum/supply_pack/misc/mining_tyrmalin
```

Comment:
Mild balance concern: Explosives are really powerful mining tools. Seismic charges are limited by them needing Mining points which you get from already mining, and compete with other mining tools. Buying explosives with cargo points might trivialize things?

I'm not against making mining even easier, of course, just a note I feel should be made.

Reply:
A hundred points is a hefty chunk of the early cargo budget, where this would matter the most, and a single (dangerous) 0,2,6 charge and 2 0,1,3 charges isn't bad

For context, a 60/60 water/potassium grenade is a 0,1,3, and is effectively useless for mining. The basic un-upgraded mining explosive is a 2,4,7.